src/scale.ts

```diff
@@ -436,15 +436,15 @@ export interface ScaleConfig<ES extends ExprRef | SignalRef> extends ScaleInvali
   zero?: boolean;
 
   /**
-   * Default framerate (frames per second) for time [`band`](https://vega.github.io/vega-lite/docs/scale.html#band) scales.
+   * Default framerate (frames per second) for [`time`](https://vega.github.io/vega-lite/docs/encoding.html#time) encoding animation [`band`](https://vega.github.io/vega-lite/docs/scale.html#band) scales.
    *
    * __Default value:__ `2`
    *
    */
   framesPerSecond?: number;
 
   /**
-   * Default animation duration (in seconds) for time encodings, except for [`band`](https://vega.github.io/vega-lite/docs/scale.html#band) scales.
```

Comment:
If this is not supported yet, we should not expose this in the spec. You can use @hidden if you still need it in the types.